README.md

```diff
@@ -77,6 +77,74 @@ $ v8r feature.geojson -c my-catalog.json
 
 This can be used to specify different custom schemas for multiple file patterns.
 
+## Configuration
+
+v8r uses CosmiConfig to search for a configuration. This means you can specify your configuration in any of the following places:
+
+- `package.json`
+- `.v8rrc`
+- `.v8rrc.json`
+- `.v8rrc.yaml`
+- `.v8rrc.yml`
+- `.v8rrc.js`
+- `.v8rrc.cjs`
+- `v8r.config.js`
+- `v8r.config.cjs`
+
+More info on how and where CosmiConfig searches for configuration can be found at https://github.com/davidtheclark/cosmiconfig#cosmiconfig
+
+Example yaml config file:
+
+```yaml
+# - One or more filenames or glob patterns describing local file or files to validate
+# - overridden by passing one or more positional arguments
+patterns: ['*json']
+
+# - Level of verbose logging. 0 is standard, higher numbers are more verbose
+# - overridden by passing --verbose / -v
+# - default = 0
+verbose: 2
+
+# - Exit with code 0 even if an error was encountered. True means a non-zero exit
+#   code is only issued if validation could be completed successfully and one or
+#   more files were invalid
+# - overridden by passing --ignore-errors
+# - default = false
+ignoreErrors: true
+
+# - Remove cached HTTP responses older than cacheTtl seconds old.
+#   Specifying 0 clears and disables cache completely
+# - overridden by passing --cache-ttl
+# - default = 600
+cacheTtl: 86400
+
+# - A custom schema catalog.
+#   This catalog will be searched ahead of any custom catalogs passed using
+#   --catalogs or SchemaStore.org
+#   The format of this is subtly different to the format of a catalog
+#   passed via --catalogs (which matches the SchemaStore.org format)
+customCatalog:
+    schemas:
+        - name: Custom Schema  # The name of the schema (required)
+          description: Custom Schema  # A description of the schema (optional)
+
+          # A Minimatch glob expression for matching up file names with a schema (required)
+          fileMatch: ["*.geojson"]
+
+          # A URL or local file path for the schema location (required)
+          # Unlike the SchemaStore.org format, which has a `url` key,
+          # custom catalogs defined in v8r config files have a `location` key
+          # which can refer to either a URL or local file.
+          location: foo/bar/geojson-schema.json
+
+          # A custom parser to use for files matching fileMatch
+          # instead of trying to infer the correct parser from the filename (optional)
+          # This property is specific to custom catalogs defined in v8r config files
+          parser: json5
+```
+
+The config file format is specified in more detail in a [JSON Schema](config-schema.json).
```

Comment:
TODO: link out to a human-readable version on https://json-schema-viewer.vercel.app once this is on `main`